observatorium/base/instance/loki/common/observatorium-loki-configmap.yaml

```diff
@@ -129,3 +129,4 @@ data:
 kind: ConfigMap
 metadata:
   name: observatorium-loki
+  namespace: opf-observatorium
```

Comment:
we should keep the hard value of namespace defined in here
either we would rely on kustomization.yaml or argocd application.

please drop this.

Reply:
I can drop this however, this means that we have to simply deploy the `configmap_patch` from the Smaug overlay as part of the kustomization, and cannot patch it. I ran into issues between this file  and the `[observatorium/overlays/moc/smaug/loki/configmaps/observatorium-loki-configmap_patch.yaml](https://github.com/operate-first/apps/pull/2253/files/a375267949230c4d7c6803ede36f1db8d401e255#diff-4d5228adcc74d5cf6bfb0c7191336f0ea69d6bb22d7e3db9c37f95e9d6a3022d) which I resolved by adding the namespace to both to make them unique resources which would allow me to build them in the observatorium/overlays/moc/smaug/loki/configmaps/kustomization.yaml with a `patchesStrategicMerge` directive.